### PR TITLE
Bringing the Salesforce builder to the masses

### PR DESCRIPTION
**Pull request by imuchnik**
This need not be private. Open sourcing...

@marcesher @Scotchester @OrlandoSoto @vickumar1981 One or more of you kind gentleman, please review.

### CODE REVIEW

File: docs/examples.md [hunk 1]
```diff
@@ -112,7 +112,30 @@ new JsJobBuilder(
...
 ```
 
-## Using MultiScm Utility 
+# Salesforce Build Job
+
+```
+
+import jenkins.automation.builders.SalesforceAntJobBuilder
+
+List developers = ['irina.muchnik@cfpb.gov', 'daniel.davis@cfpb.gov']
```
Comment (marcesher) [inline docs/examples.md]: let's use example.com emails in all our examples / docs

File: src/main/groovy/jenkins/automation/builders/SalesforceAntJobBuilder.groovy [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+        def job = new BaseJobBuilder(
+                name: name,
+                description: description,
+                emails: emails
+        ).build(factory);
+
+        job.with {
+            logRotator {
+                numToKeep(365)
```
Comment (marcesher) [inline src/main/groovy/jenkins/automation/builders/SalesforceAntJobBuilder.groovy]: Why the override?

File: src/main/groovy/jenkins/automation/builders/SalesforceAntJobBuilder.groovy [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            }
+            scm {
+                git {
+                    remote {
+                        url(repoUrl)
```
Comment (marcesher) [inline src/main/groovy/jenkins/automation/builders/SalesforceAntJobBuilder.groovy:48]: Do we know it'll always be a single URL? Seems like this should be wrapped in an if statement, to allow for flexibility, i.e. if folks do have multiple repos, they can skip repoUrl and use multiscm directly

File: src/main/groovy/jenkins/automation/builders/SalesforceAntJobBuilder.groovy [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                    }
+                }
+            }
+            steps {
+                ant {
+                    target(antTaskName)
```
Comment (marcesher) [inline src/main/groovy/jenkins/automation/builders/SalesforceAntJobBuilder.groovy]: recommend using `targets` instead

File: src/main/groovy/jenkins/automation/builders/SalesforceAntJobBuilder.groovy (hunk 2 above)
Comment (imuchnik) [inline src/main/groovy/jenkins/automation/builders/SalesforceAntJobBuilder.groovy]: Good catch. Will remove.

File: docs/examples.md (hunk 1 above)
Comment (imuchnik) [inline docs/examples.md]: Sure. No objections.

File: src/main/groovy/jenkins/automation/builders/SalesforceAntJobBuilder.groovy (hunk 3 above)
Comment (imuchnik) [inline src/main/groovy/jenkins/automation/builders/SalesforceAntJobBuilder.groovy:48]: Trying with one right now. Will change if needed later. Not sure if SF supports development across multiple repos.

File: src/main/groovy/jenkins/automation/builders/SalesforceAntJobBuilder.groovy (hunk 4 above)
Comment (imuchnik) [inline src/main/groovy/jenkins/automation/builders/SalesforceAntJobBuilder.groovy]: done.

**Pull request merged**